opennms-webapp/src/main/java/org/opennms/web/controller/alarm/AcknowledgeAlarmByFilterController.java

```diff
@@ -61,7 +62,10 @@
     private AlarmRepository m_webAlarmRepository;
     
     private String m_redirectView;
-    
+
+    private RedirectRestricter redirectRestricter = RedirectRestricter.builder()
```

Comment:
No allowed redirects? Is this intentional?

Reply:
yes, I didn't find any link with a redirect pointing to this controller (although I might have missed one). The alternative would be to remove the redirect parameter completely from this controller (would result in the same thing but viewer code). I wonder why we have the redirect parameter if we don't use it:
Either this is just a copy and paste leftover or
I didn't find the link...

Reply:
TODO: Patrick add as comment in code

Reply:
done.